git/tests/test.pester.ps1

```diff
@@ -0,0 +1,33 @@
+param (
+    [Parameter()]
+    [string]$PackageIdentifier = $(throw "Fully qualified package identifier must be given as a parameter.")
+)
+
+$_origin,$_package,$_version,$_timestamp = $PackageIdentifier.split('/')
+Describe "$_package version $_version" {
+    Context "It" {
+        ${_installation_directory}="C:/hab/pkgs/$PackageIdentifier"
+        It "is installed in the '${_installation_directory}' directory" {
+            "${_installation_directory}" | Should -Exist
```

Comment:
In the past week or two there was PR feedback that it's best not to do these package directory existence tests.
https://github.com/habitat-sh/core-plans/pull/2624 removed such tests from the design doc.

I've no objection to merging these tests, just an FYI that they're not standard anymore.

I think this test can also spuriously pass if `$PackageIdentifier` isn't set because it will test if `C:/hab/pkgs/` exists, maybe.

Reply:
Hi @james-stocks.  Thanks for feedback.  I will be removing this directory check shortly according to #2624 and https://github.com/habitat-sh/core-plans/issues/2678